src/main/java/github/scarsz/discordsrv/listeners/PlayerJoinLeaveListener.java

```diff
@@ -102,7 +102,7 @@ public void onPlayerJoin(PlayerJoinEvent event) {
         }
     }
 
-    @EventHandler //priority needs to be different to MONITOR to avoid problems with permissions check when PEX is used
+    @EventHandler(priority = EventPriority.LOWEST) //priority needs to be different to MONITOR to avoid problems with permissions check when PEX is used, is at lowest so that it executes before VanishNoPacket's player leave listener and is able to see whether the player is vanished
```

Comment:
```suggestion
    @EventHandler(priority = EventPriority.LOWEST) //priority needs to be different to MONITOR to avoid problems with permissions check when PEX is used, it is at lowest so that it executes before VanishNoPacket's player leave listener and is able to see whether the player is vanished
```